lib/controllers/frontend/spree/apple_pay_domain_verification_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Spree
+  class ApplePayDomainVerificationController < StoreController
+    skip_before_action :set_current_order, only: :show
+    include Spree::Core::ControllerHelpers
+
+    def show
+      gateway = Gateway.active.find_by!(type: 'Spree::Gateway::StripeApplePayGateway')
```

Comment:
Can we use `gateway = Spree::Gateway::StripeApplePayGateway.active.first` here or something similar?
I don't like the idea of hardcoding class name into controller

Reply:
fixing